b3lb/loadbalancer/settings.py

```diff
@@ -93,6 +94,8 @@
 
 ROOT_URLCONF = 'loadbalancer.urls'
 
+DEFAULT_FILE_STORAGE = 'db_file_storage.storage.DatabaseFileStorage'
```

Comment:
Shoud be removed since we provide the `storage=` kwargs in the Assets model, shouldn't it?